lib/meta_inspector/url.rb

```diff
@@ -5,7 +5,7 @@ class URL
     attr_reader :url
 
     def initialize(initial_url, options = {})
-      options        = defaults.merge(options)
+      options        = self.class.defaults.merge(options)
```

Comment:
Not sure why this is has been changed here, why the need for `self`?

Reply:
Because I wanted to use the same `default` method in `def self.absolutify(...)` without having to instantiate the URL object or without duplicating code. Maybe there is a better way...